src/Fixer/CastNotation/ShortScalarCastFixer.php

```diff
@@ -42,7 +42,7 @@ public function isCandidate(Tokens $tokens): bool
 
     protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
     {
-        static $castMap = [
+        $castMap = [
```

Comment:
i wonder if we can somehow indicate that it shouldn't be modified during runtime.

Reply:
should we move to private class-level const?
https://3v4l.org/vjlmm#v8.3.23